lua/plenary/curl.lua

```diff
@@ -194,7 +151,7 @@ end
 ------------------------------------------------------------
 
 parse.response = function(lines, dump_path, code)
-  local headers = util.readfile(dump_path)
+  local headers = P.readlines(P.new(dump_path))
```

Comment:
@tjdevries  :smile: it would be nice if path.new is not needed here. and instead path module function would accept a string and handle it if path object is not provided.